### DISCUSSION TITLE
在发布时找不到对应的控件

### DISCUSSION THREAD

**Discussion by Yoongool** [HandyOrg/HandyControl#1335] [General]
例如：在xaml中 引入了 xmlns:hc="https://handyorg.github.io/handycontrol"，可以正常编译调试，但是在发布程序时提示
错误	MC3074	XML 命名空间“https://handyorg.github.io/handycontrol”中不存在标记“SearchBar”。 

本地环境；vs2019

**Comment by NaBian** [comment]
缺少必要的上下文，无法给出解决方案。

**Comment by Helevn** [comment]
同问:

在项目中执行 dotnet build命令
报错：XML 命名空间“https://handyorg.github.io/handycontrol”中不存在标记“TransitioningContentControl”。

我已经再程序中应用，而且可以使用VS2022运行，但是build的时候就会出现这个错误。

我使用的框架SDK是.NET8.0，
